integration_tests/e2e/goal/createAndEditGoal.cy.ts

```diff
@@ -0,0 +1,183 @@
+import Page from '../../pages/page'
+import AddStepPage from '../../pages/goal/AddStepPage'
+import AddNotePage from '../../pages/goal/AddNotePage'
+import ReviewPage from '../../pages/goal/ReviewPage'
+import OverviewPage from '../../pages/overview/OverviewPage'
+import CreateGoalPage from '../../pages/goal/CreateGoalPage'
+import AuthorisationErrorPage from '../../pages/authorisationError'
+
+context('Create and edit a goal', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignInAsUserWithEditAuthority')
+    cy.task('stubAuthUser')
+    cy.task('stubGetHeaderComponent')
+    cy.task('stubGetFooterComponent')
+    cy.task('stubPrisonerList')
+    cy.task('stubCiagInductionList')
+    cy.task('stubActionPlansList')
+    cy.task('getPrisonerById')
+    cy.task('stubGetShortQuestionSetCiagProfile')
+    cy.task('getActionPlan')
+    cy.task('stubLearnerProfile')
+    cy.task('stubLearnerEducation')
+  })
+
+  it('should be able to navigate to edit goal description page given user has clicked change from the review page', () => {
```

Comment:
I think you can remove this scenario, and the next one, because:
1. your 3rd and 4th scenarios (where I commented that it was an excellent test) do everything that is in this test, plus crucially the actual submission of the edit back to the Review page and asserts the changes are made 👍